java-app/manifest.yml

```diff
@@ -1,7 +1,7 @@
 ---
 applications:
 - name: solace-sample-java-app
-  memory: 512M
+  memory: 1G
```

Comment:
This seems way too high.

Reply:
We tested on more recent CF releases with higher buildpacks and increasing the memory was only way to fix it.

Reply:
Seems unreasonably high. But for now, that's not a specific concern of this review.